src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedButton: string;
```

Comment:
Btw, it should be called pressedKey
![image](https://user-images.githubusercontent.com/96409563/201057533-97b50279-33c7-4d2a-8ee2-117e6e42c005.png)
```suggestion
  pressedKey: string;
```

Reply:
renamed